lib/NonDetMonadLemmaBucket.thy

```diff
@@ -42,15 +42,11 @@ lemma return_bindE:
   "isRight v \<Longrightarrow> return v >>=E f = f (theRight v)"
   by (cases v; clarsimp simp: return_returnOk)
 
-lemma list_case_return: (* FIXME lib: move to Lib *)
+lemma list_case_return: (* not in Lib, because "return" is not in scope there *)
```

Comment:
sanity check: `Monads` is part of what's going to get factored out of `lib`, right? So NonDetMonadLemmaBucket, despite its name, shouldn't go there?

Reply:
Yes, Monads gets moved out, but NonDetMonadLemmaBucket stays in. It's mostly left over as convenience to not disturb the l4v proof context too much, but we could actually try to eliminate it completely. I'd vote for doing that later, though.